### PR TITLE
[Merged by Bors] - Add Digital Extinction

### PR DESCRIPTION
**Pull request by Indy2222**

### CODE REVIEW

File: Apps/Games/digital-extinction.toml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name = "Digital Extinction"
+description = "3D real-time strategy game"
```
Comment (alice-i-cecile) [inline Apps/Games/digital-extinction.toml:2]: I won't block on this, but I think you'll do better with a more detailed and evocative description :) This is your genre, but what's your hook?

Comment (Indy2222) [reply Apps/Games/digital-extinction.toml]: You are right. I've tried something better.

**Comment by alice-i-cecile** [comment]
bors try

**Pull request closed without merging**